src/main/java/org/codehaus/mojo/exec/EnvStreamConsumer.java

```diff
@@ -28,10 +32,21 @@ public void consumeLine( String line )
 
         if ( this.startParsing )
         {
-            String[] tokens = StringUtils.split( line, "=" );
+            String[] tokens = StringUtils.split( line, "=", 2 );
             if ( tokens.length == 2 )
             {
                 envs.put( tokens[0], tokens[1] );
+            } else {
+                // Don't hide an environment variable with no value e.g. APP_OVERRIDE=
+                if ( line.trim().endsWith("=") )
+                {
+                    String trimmed = line.trim();

```

Comment:
Could we move this to before condition to have `trim` applied only once?

Reply:
Good point. Done.